src/xdocs/config_whitespace.xml

```diff
@@ -1060,6 +1068,15 @@ import static java.math.BigInteger.ZERO;
     &lt;property name=&quot;allowLineBreaks&quot; value=&quot;false&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>
+         If the annotation is between the type and the array, the check will skip:
+        </p>
+        <source>
+public void a(final char @AnnotationAfterTest [] a) {}//Correct
```

Comment:
please put there `@NotNull` to be close to real life.
All cases from below please remove, they are too syntactic.
"//Correct" ==> "//No violation"

Reply:
fixed

Reply:
@sergileon 
not fixed, please read attentively - you need to keep only one example:
```java
public void a(final char @NotNull [] a) {} // No violation
```

Reply:
@Vladlis  fixed